app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,32 @@
+class OrdersController < ApplicationController
+  def show
+    @order = Order.find_by(id: session[:order_id])
+    render :show
```

Comment:
The final `render` line here isn't strictly necessary but there is also no harm in it being here explicitly.

It is just a rails magic thing where if undeclared rails will try to find the view that matches the name of the controller and method.